renovate.json

```diff
@@ -0,0 +1,15 @@
+{
+  "baseBranches": [
+    "develop"
+  ],
+  "gradle": {
+    "ignorePaths": [
+      "gradle/libs.versions.toml",
+      "benchmark/**",
+      "dsl/**",
+      "query-model/**",
+      "render/**",
+      "support/**"
+    ]
```

Comment:
If it uses ignorePaths, I'll have to worry about the renovate setting every time a new file is added. Is there a way to do it with include path? 

https://docs.renovatebot.com/configuration-options/#includepaths